src/test/kotlin/fi/metatavu/timebank/api/test/functional/impl/ApiTestBuilderResource.kt

```diff
@@ -25,10 +25,10 @@ abstract class ApiTestBuilderResource<T, A>(
      */
     override fun getApiClient(): ApiClient {
         ApiClient.builder
-            .connectTimeout(Integer.MAX_VALUE.toLong(), TimeUnit.SECONDS)
-            .writeTimeout(Integer.MAX_VALUE.toLong(), TimeUnit.SECONDS)
-            .readTimeout(Integer.MAX_VALUE.toLong(), TimeUnit.SECONDS)
-            .callTimeout(Integer.MAX_VALUE.toLong(), TimeUnit.SECONDS)
+            .connectTimeout(600, TimeUnit.SECONDS)
```

Comment:
What was the reason for this change?

Reply:
Integer.MAX_VALUE.toLong gives a value large enough to cause an error, so i chanced it to 600 second.